packages/imperative/src/profiles/src/abstract/AbstractProfileManager.ts

```diff
@@ -200,12 +200,12 @@ export abstract class AbstractProfileManager<T extends IProfileTypeConfiguration
         if (parms.loadCounter != null) {
             this.mLoadCounter = parms.loadCounter;
         }
-        this.mLogger = isNullOrUndefined(parms.logger) ? this.mLogger : parms.logger;
+        this.mLogger = parms.logger == null ? this.mLogger : parms.logger;
         this.mProfileType = parms.type;
         this.mProfileRootDirectory = parms.profileRootDirectory;
         this.mProfileTypeConfigurations = parms.typeConfigurations;
         this.mProductDisplayName = parms.productDisplayName;
-        if (isNullOrUndefined(this.profileTypeConfigurations) || this.profileTypeConfigurations.length === 0) {
+        if (this.profileTypeConfigurations == null || this.profileTypeConfigurations.length === 0) {
```

Comment:
Similar to the previous comment...

what about using `?.` 😋 

There are edge cases where `?.` may not be desired since it can technically change the behavior. For example, `if (a != null && !a.booleanValue)` is not the same as `if (!a?.booleanValue)` since the condition will evaluate to true if `A` is undefined 😋

Reply:
Same as previous comment.